mock-thunderhead/src/main/java/com/sequenceiq/thunderhead/grpc/service/auth/MockUserManagementService.java

```diff
@@ -878,7 +877,6 @@ public void getAccount(GetAccountRequest request, StreamObserver<GetAccountRespo
             builder.addEntitlements(createEntitlement(CDP_CCM_V2_JUMPGATE));
         }
         if (verticalScaleEnabled) {
-            builder.addEntitlements(createEntitlement(CDP_CB_AWS_VERTICAL_SCALE));
```

Comment:
If we have public api which checks this entitlement, it will cause issue while running CB locally since entitlement will not be assigned to the user.

Reply:
Neither the CLI nor the UI uses this anymore